test/reporter-stdout.js

```diff
@@ -0,0 +1,21 @@
+/*globals QUnit:true*/
+/*jshint node:true*/
+var QUnit = require( "../dist/qunit" );
+
+// Options: { output: "minimal" || "verbose" }
+QUnit.stdout();
+
+// Load QUnit tests
+require( "./logs" );
```

Comment:
That `var QUnit` shouldn't be accessible in the scope of any other files required here. Its a local variable, not made global in any way.